README.md

```diff
@@ -84,12 +84,45 @@ You can use a combination of the next env variables to override your profile set
 - `MCLI_ORG_ID`
 - `MCLI_OPS_MANAGER_URL`
 
-### ZSH Completion (experimental)
-Add the following to your `.zshrc` file
+### Shell Completions
+
+If you install via [homebrew](#hombrew-on-macos) there's nothing else to do. 
+For other installations please refer to your preferred shell instructions.
+
+#### Bash
+
+```bash
+$ source <(mongocli completion bash)
+```
+
+To load completions for each session, execute once:
+
+Linux:
+```bash
+mongocli completion bash > /etc/bash_completion.d/mongocli
+```
+  
+macOS:
+```bash
+mongocli completion bash > /usr/local/etc/bash_completion.d/mongocli
+```
+
```

Comment:
I noticed you have powershell in the usage, but no note here about it.

Powershell has the argument completer module, but I haven't tried to set it up. https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/register-argumentcompleter?view=powershell-6

Reply:
cobra doesn't have documentation on how this works an since I don't have powershell at hand to test it I'd let the community come back with the info